pkg/term/term.go

```diff
@@ -127,6 +127,7 @@ func handleInterrupt(fd uintptr, state *State) {
 	go func() {
 		_ = <-sigchan
 		RestoreTerminal(fd, state)
+		// why is this here?
```

Comment:
Forgot I left that in there. Should I nuke the exit? I don't think it's a problem that we restore the terminal twice.